firefox-ios/Client/Frontend/Browser/Search/SearchViewController.swift

```diff
@@ -845,15 +838,10 @@ class SearchViewController: SiteTableViewController,
         cell.accessoryView = nil
     }
 
-    func append(_ sender: UIButton) {
-        let buttonPosition = sender.convert(CGPoint(), to: tableView)
-        if let indexPath = tableView.indexPathForRow(
-            at: buttonPosition
-        ), let newQuery = viewModel.suggestions?[indexPath.row] {
```

Comment:
not sure why we needed to do this check as opposed to passing in the string, but didn't see any issues from testing 🤔 .